src/tests/utils/utils.cairo

```diff
@@ -36,6 +36,29 @@ fn sum_u256_array(mut arr: Span<u256>) -> u256 {
     sum
 }
 
+fn create_default_option_amount_array(mut len: u32, mut options_available: u256) -> Array<u256> {
+    let mut arr: Array<u256> = array![];
+    let mut index: u32 = 0;
+    while (index < len) {
+        arr.append(options_available);
+        index += 1;
+    };
+    arr
+}
+
+// Sum the total amount paid for multiple bids.
+fn get_total_bids_amount(mut bid_prices: Span<u256>, mut bid_amounts: Span<u256>) -> u256 {
```

Comment:
this is ok for now, but just for reference if we ever decide to standardize our helpers, this is the same as the dot product of two arrays, and Alexandria has a helper for this here: [https://github.com/keep-starknet-strange/alexandria/blob/ef3292bbd9b35218dea18876373a31bf7f18934e/packages/linalg/src/dot.cairo#L10](url)